### DISCUSSION TITLE
Wrong page title in finish mama project

### DISCUSSION THREAD

**Discussion by Vivioza5** [open-tutorials/cypress#34] [Ideas]
# 🐞 Bugreport tutorial **Finish mama project**

 * **Number:** 2
 * **Name:** Spelling error in the title of the page
 * **Date:** 05/02/2023
 * **Project:** Open tutorials
 * **Part:** Title
 * **Severity:** Normal
 * **Priority:** Medium

 # Table of contents
 1. Behaviour
 2. Precondition
 3. Steps
 4. Actual result
 5. Expected result
 6. Possible causes
 7. Correction

 # 1. Behaviour
 * ** Device:** Any devices
 * **OS:** Any OS
 * **Screen:** any screens
 * **Browser:** Any browser
 * **Network parameters:** Any networks

# 2. Precondition
 1. Open [tutorial](https://www.epic1h.com/finish_mama_project) on computer

# 3. Steps
 [Scring recording](https://www.awesomescreenshot.com/video/14552628?key=312954dff5a0872850218e4d553bdc34)

2. Check title of page

# 4. Actual result
 Title is "Туториал: как устроен Cypress внутри"

 # 5. Expected result
 Title is "Туториал: заканчиваем мама проект"

 # 6. Possible causes
 Mistake

 
 # 7. Correction
 Filled in by the developer in charge.
 
 * ** Responsible:**
 * **Date of correction:**
 * **Link to patch or commit:**
 * **Release link:**
 
 ## Factual causes

**Comment by Vladimir-Tarasov** [comment]
@Vivioza5 thanks. will check

**Reply by breslavsky** [reply]
@Vladimir4932 where is result here? Could you reproduce it?

**Reply by Vladimir-Tarasov** [reply]
<img width="600" alt="Screenshot 2023-02-13 at 16 06 58" src="https://user-images.githubusercontent.com/92365254/218494895-ee636df8-5b7e-40ea-92f6-a3d08ecbe3e6.png">
